unfetter-discover/unfetter
# Centralize proxy in Ansible

Fixes unfetter-discover/unfetter#1457

The HTTPS_PROXY_URL variable has been moved out if the processor, api, and socket server roles and into a centralized Ansible file (all.yml) and can be overridden at the hosts level.  Default proxy setting is empty as before.

### To test:
1. Pull this branch
2. Introduce an override file (e.g. dev.yml for deploying in development mode) in the host_vars directory under unfetter/ansible
3. Add `https_proxy_url: <some url>` to dev.yml
4. Start the stack
5. Confirm that the proxy URL above is shown in the debug task variable listing
